### DISCUSSION TITLE
Does Netbeans support DAP protocol?

### DISCUSSION THREAD

**Discussion by magbroxwt** [apache/netbeans#3934] [Q&A]
In [https://microsoft.github.io/debug-adapter-protocol/implementors/tools/](url) I don't see Netbeans.
But I seem to see that DAP client have been used in  [https://github.com/apache/netbeans/pull/3119/files/a9e18c2a6252ceda5947586c92c6df470c8b0894#](url) by importing org.netbeans.api.debugger or org.eclipse.lsp4j.debug?
Maybe Netbeans use lsp4j which is defined as "Java binding for the Language Server Protocol and the Debug Adapter Protocol" to support DAP itself?

**Comment by matthiasblaesing** [comment]
You are looking at the lsp-server part. To my knowledge there is noone working on it. You couldbe the first ;-)
